server/models/queries.js

```diff
@@ -51,8 +55,8 @@ queries.updateReview = `UPDATE reviews SET title = $2, description = $3 WHERE _i
 queries.deleteReview = `DELETE FROM reviews WHERE _id = $1`;
 
 //Users
-queries.createUser = `INSERT INTO users (first_name, last_name, full_name, username, email, password, is_landlord) 
-VALUES ($1, $2, $3, $4, $5, $6, $7)
+queries.createUser = `INSERT INTO users (first_name, last_name, full_name, username, email, company, password, is_landlord) 
```

Comment:
confirm order of parameters with `userController.createUser`. Also revise based on decision to keep `company` and `is_company` in the same table

Reply:
fixed!